responsibleai/tests/causal/test_causal_manager.py

```diff
@@ -106,6 +107,129 @@ def test_causal_train_test_categories(self, parks_data):
                         skip_cat_limit_checks=True,
                         upper_bound_on_cat_expansion=50)
 
+    def test_causal_manager_global_cohort_effects(self, housing_data):
+        train_df, test_df, target_feature = housing_data
+
+        manager = CausalManager(train_df, test_df, target_feature,
+                                ModelTask.REGRESSION, None)
+        manager.add(['AveRooms'])
+        manager.compute()
+
+        id = manager.get()[0].id
+        X_test = test_df.drop(target_feature, axis=1)
+        causal_data = manager.request_global_cohort_effects(id, X_test)
+
+        assert isinstance(causal_data, CausalData)
```

Comment:
I see these asserts repeated throughout the tests below, in lines 122-126, 159-163, 188-192, except for the last line each case - perhaps consider refactoring into helper validation method